README.md

```diff
@@ -1,16 +1,42 @@
 # Facebook Group Posts Scraper 
-​​
+
 Facebook Group Posts Scraper is a package which is used for scraping facebook group posts by their group ids.
-​
+
 ## Getting Started - Installation
-​
-`npm i facebook-group-posts-scraper -g --unsafe-perm`
-​
+
+```sh
+npm i facebook-group-posts-scraper -g --unsafe-perm
+fgps init
+```
 ## Usage
+
+To scrape posts from group accessible by URL facebook.com/groups/some_group/:
+
+```sh
+fgps --group-ids some_group --output /some/output/dir
+```
+
+For many groups at once:
+```sh
+fgps --group-ids some_group1,some_group2,some_group3 --output /some/output/dir
+```
+
+App will output hundreds of `i=xxx` lines on the screen and finally a data files will be placed in specified output dir.
+
+```
+i=1
+i=2
+...
+i=1082
+i=1083
+...
+```
```

Comment:
Hi @gakowalski ! 
 
 Hope you're doing great! Appreciate your contribution! I just merged #21 to fix the flooding on standard output. It would be great if you can remove from the readme :)

Reply:
Hi, I've removed the lines about `i=xxx` flood. I believe you could do some command line switch to enable those debug messages as they can be useful - they are some kind of indicator that application works.

Reply:
Completely agree! We're currently working on a TypeScript version of the project and also we've changed the project's name to `fbjs`. To extends the project's scope with an entire Facebook automation 😎 Any contributions on the `typescript-migration` branch will be more than welcome !